src/Http/Controllers/SignedCheckoutController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rapidez\Core\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use Rapidez\Core\Models\Quote;
+
+class SignedCheckoutController
+{
+    public function __invoke(Request $request)
+    {
+        if (! $request->hasValidSignature() || ! Cache::has('checkout-' . $request->get('key'))) {
+            return redirect(config('rapidez.magento_url'), 301);
+        }
+
+        $data = Cache::get('checkout-' . $request->get('key'));
```

Comment:
So we store something when creating the signed route and when navigating to it we'll get it. What if in the meantime the cache will be flushed?

Reply:
Can't we store it somewhere persistent?

Reply:
I've added docs on using the standalone checkout, and configuring more persistent storage: https://github.com/rapidez/docs/pull/74